src/components/inputs/localized-rich-text-input/editor.js

```diff
@@ -146,14 +146,24 @@ const Editor = props => {
                   )
                 );
               })()}
-              {renderToggleButton && isOpen && (
+              {renderToggleButton && (
                 <React.Fragment>
                   <LeftColumn />
                   <RightColumn>
                     <FlatButton
                       onClick={toggle}
-                      label={intl.formatMessage(messages.collapse)}
-                      icon={<AngleUpIcon size="small" />}
+                      label={
+                        isOpen
+                          ? intl.formatMessage(messages.collapse)
+                          : intl.formatMessage(messages.expand)
+                      }
+                      icon={
+                        isOpen ? (
```

Comment:
Sometimes I wonder if doing a 

```js
const ToggleButtonIcon = props => props.collapsed ? <AngleDownIcon size="small" /> : <AngleUpIcon size="small" />; 
```

and then

```js
icon={<ToggleButtonIcon collapsed={!props.isOpen} />}
```

is just overly eager or now. 🤷‍♂